packages/models-library/src/models_library/api_schemas_directorv2/clusters.py

```diff
@@ -113,7 +113,7 @@ class ClusterDetailsGet(ClusterDetails):
 
 
 class ClusterCreate(BaseCluster):
-    owner: GroupID | None
+    owner: GroupID | None = None  # type: ignore[assignment]
```

Comment:
actually I did not see the `= None` in my last check. only the ignore.
why is this changed to optional? another test?

Reply:
because of the below attached example. One has the owner field one does not have it expiclty specified. So it implies it has to be None by default